test/integration/simple-template/controls/project.rb

```diff
@@ -0,0 +1,7 @@
+title "Test simple template"
+
+message = attribute("message")
```

Comment:
Maybe some comments here letting the user know what the attributes are and how they relate to terraform?